client/src/components/layouts/LayoutEpsilon/cardSwitcher.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from "react";
+
+class CardSwitcher extends Component {
+  state = {};
+  render() {
+    const { cards, cardName, changeCard, hyperCard } = this.props;
+    //const iconName ="";
+    const currentCard = cards.find(c => c.name === cardName);
```

Comment:
We need to have a way to default to the first card in the event that we can't find the card with the card name. That's why it is currently crashing. Change this line like so:
```suggestion
    const currentCard = cards.find(c => c.name === cardName) || cards[0];
```
That will make it so if it can't find the card, it just uses the first one.